browser/BUILD.gn

```diff
@@ -48,6 +48,8 @@ source_set("browser_process") {
     "brave_browser_process_impl.h",
     "brave_content_browser_client.cc",
     "brave_content_browser_client.h",
+    "brave_cosmetic_resources_tab_helper.cc",
```

Comment:
this should go in `brave/browser/cosmetic_filters` and be a separate target

Reply:
no longer an issue